.changes/next-release/Feature-ed68db23-5ec8-42ba-8a31-8dfd4a826f92.json

```diff
@@ -0,0 +1,4 @@
+{
+    "type": "Feature",
+    "description": "Upgrade amazon-states-language-service dependency version to 1.4.0"
```

Comment:
This description doesn't have meaning to customers. What will be different for them when they obtain the version of the toolkit containing this changelog entry?

Reply:
Will change it to something more meaningful for the customer

Reply:
this old json file needs to be deleted too

Reply:
it has been removed :)